.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,10 +13,19 @@ Note that to get help, you need to run the latest version.
 
 **Where is the issue?**
  * [ ] M.IM.JsonWebTokens
+ * [ ] M.IM.Protocols
  * [ ] M.IM.Protocols.OpenIdConnect
+ * [ ] M.IM.Protocols.SignedHttpRequest
+ * [ ] M.IM.Protocols.WsFederation
  * [ ] M.IM.Tokens
- * [ ] M.IM.Tokens.Jwt
+ * [ ] S.IM.Tokens.Jwt
```

Comment:
Could we put the items in lexicographical order?

Reply:
yes. will fix!

Reply:
#1760